microsoft-edge/toc.yml

```
@@ -1000,9 +1000,10 @@
                   href: webview2/how-to/javascript.md
                   displayName: ExecuteScriptAsync API, Use JavaScript in WebView for extended scenarios  # old title
 
-                # - name: Call native-side code from web-side code
-                #   href: webview2/how-to/add-host-object-to-script.md
-                #   displayName: AddHostObjectToScript API
+              # - name: Call native-side code from web-side code
+                - name: Using Host Objects in WebView2
+                  href: webview2/how-to/hostobject.md
+                  displayName: AddHostObjectToScript API
```

```suggestion
                - name: Call native-side code from web-side code
                  href: webview2/how-to/hostobject.md
                  displayName: AddHostObjectToScript, host objects
```
This title resides at a higher level than underlying technical implementation.  "Using host objects" expresses at the lowest level of technical implmentation.  I believe the TOC redesign PR discussion reached consensus that this higher-level title, "Call native-side code from web-side code", in a coordinated set of 2-4 article titles, would be this pattern instead.  

If change the TOC title here, then also in the .md file, change the top-of-page title (in _two_ lines: the "title: " yaml field that controls the browser-tab title, and in the `#` h1 line).
The rule is: the TOC title is allowed to delete words compared to the top-of-page title, but is not allowed to use different words.  That rule is to prevent confusion & surprises for authors & readers during navigation.